### PR TITLE
Fixed errcheck issues in server/channels/app/migrations.go

### PR DESCRIPTION
**Pull request by Aryakoste**
#### Ticket Link

Fixes https://github.com/mattermost/mattermost/issues/29031
Jira https://mattermost.atlassian.net/browse/MM-61443
#### Release Note

```release-note
NONE
```

### CODE REVIEW

File: server/channels/app/migrations.go [hunk 1]
```diff
@@ -105,7 +105,9 @@ func (a *App) SetPhase2PermissionsMigrationStatus(isComplete bool) error {
 }
 
 func (a *App) DoEmojisPermissionsMigration() {
```
Comment (hanzei) [inline server/channels/app/migrations.go:107]: Let's return an error and check that. That ensures the error gets surfaced.

**Review by hanzei — CHANGES_REQUESTED** [review]
Thanks for the PR :+1:

**Comment by Aryakoste** [comment]
@hanzei Changes done

File: server/channels/app/migrations.go [hunk 2]
```diff
@@ -163,7 +166,9 @@ func (s *Server) doEmojisPermissionsMigration() error {
 }
 
 func (a *App) DoGuestRolesCreationMigration() {
```
Comment (hanzei) [inline server/channels/app/migrations.go:168]: I would like to see the error getting returned here, but it's not a blocker.

**Review by hanzei — APPROVED** [review]

**Review by fmartingr — APPROVED** [review]

**Review by hanzei — APPROVED** [review]

**Comment by hanzei** [comment]
Thanks @Aryakoste :tada:

**Pull request merged**